### ISSUE TITLE
Retractable bridges.

### ISSUE THREAD

**Issue by MikeyCarter** [orphu/mcdungeon#299]
Was looking at this and thought... retractable bridges.  (like your retractable gates) Maybe have it so the button's on the opposite wall so you'd have to hit with an arrow to activate.

![2015-04-20_17 58 35](https://cloud.githubusercontent.com/assets/1325499/7241636/b035db10-e788-11e4-879c-2bb014a8bae0.png)

**Comment by sshipway** [comment]
I like this one.  Might be a bit complex on cases like this with an angled bridge though.  
I hadn't realised you can hit a button using an arrow...
Makes it feel a bit like that scene in the Death Star with Luke+Leia and the retractable bridges...

**Comment by orphu** [comment]
Yeah, neat. Hiding the mechanism could be a challenge. I think you'd need a command block per bridge "leg."

Maybe a special case room that would appear only of the bridge was straight?
